src/DecayManager.cc

```diff
@@ -96,11 +99,27 @@ void DecayManager::RegisterDistribution(const string name,
 
 vector<vector<double> >* DecayManager::GetDistribution(const string name) {
   if (registeredDistributions.count(name) == 0) {
-    throw std::invalid_argument("Distribution not registered.");
+    throw std::invalid_argument("Transition not registered.");
```

Comment:
What's the reason for changing this to transition? The code deals here with a specific energy distribution, of which there can be several for a transition

Reply:
Yes, it's a wrong copy/paste, I will correct it.